packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -622,6 +716,196 @@ You can see additional information about your Validator, including uptime score,
 celocli validator:show $CELO_VALIDATOR_RG_ADDRESS
 ```
 
+## Registering as a Validator (Non-ReleaseGold)
+
+If you are new to validating on Baklava, then you most likely need to be following the Non-ReleaseGold steps for getting your validator up and running. ReleaseGold is done in the previous section and only concerns those who are generating accounts with ReleaseGold smart contract.
+
+### Register the Accounts
+
+You've now done all the infrastructure setup to get a validator and proxy running. The cLabs team will review your submission to receive funds and send you 12,000 testnet CELO to each of your Validator and Validator Group account addresses. These funds have no real world value but will allow you to submit transactions to the network via celocli and put up a stake to register as a validator and validator group.
```

Comment:
Suggest clarifying `12,000 Baklava testnet CELO`

Reply:
wrap `celocli` with code formatting and suggest making it link to `celocli` repo